src/amo/containers/DetailPage.js

```diff
@@ -19,17 +20,20 @@ export class DetailPageBase extends React.Component {
 
 function mapStateToProps(state, ownProps) {
   const { slug } = ownProps.params;
+  const addon = state.addons[slug];
   return {
-    addon: state.addons[slug],
+    addon,
     slug,
+    versionDetails: addon && state.version[addon.current_version.id] ?
+      state.version[addon.current_version.id] : state.version,
```

Comment:
I think it's because of how my reducers are returning values which isn't great. Agreed, this should be fixed.